src/coreclr/vm/method.hpp

```diff
@@ -2023,16 +2023,23 @@ class MethodDesc
 
 private:
     PCODE PrepareILBasedCode(PrepareCodeConfig* pConfig);
+#ifdef FEATURE_TIERED_COMPILATION
+    PCODE GetPrecompiledCode(PrepareCodeConfig* pConfig, bool shouldCountCalls);
+    PCODE JitCompileCode(PrepareCodeConfig* pConfig, bool shouldCountCalls);
+    PCODE JitCompileCodeLockedEventWrapper(PrepareCodeConfig* pConfig, JitListLockEntry* pEntry, bool shouldCountCalls);
+    PCODE JitCompileCodeLocked(PrepareCodeConfig* pConfig, JitListLockEntry* pLockEntry, bool shouldCountCalls, ULONG* pSizeOfCode, CORJIT_FLAGS* pFlags);
```

Comment:
The `JitCompileCode*` methods don't need `shouldCountCalls` passed in, they would only call `PrepareCodeConfig::ShouldCountCalls()` if doing a tier 0 JIT, which it won't be doing for `UnmanagedCallersOnly` because tiering / call counting has already been disabled.

Reply:
After the other suggestion below